docs/en/UI/Blazor/Page-Progress.md

```diff
@@ -1,3 +1,62 @@
 # Blazor UI: Page Progress
 
-TODO
\ No newline at end of file
+Page progress is the newest component introduced with Blazorise and ABP Framework. It is used to show scoped page progress on top of the page and to indicate to the user that currently a long running process is in the work. By default you don't need to do anything to show the progress indicator, as all the work is done automatically by the ABP Framework internals. This means that all calls to the ABP backend(through Rest API) will activate page progress and show the loading indicator.
```

Comment:
>Page progress is the newest component introduced with Blazorise and ABP Framework

If we write this here, then we will need to change this document when we introduce a new component with Blazorise or ABP Framework.

Reply:
I just made a few changes in the doc: https://github.com/abpframework/abp/commit/79575ff040747def938f1670f4926afca5b0b988 ;)